bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/update/update.xml

```diff
@@ -0,0 +1,589 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="guntamatic:biostar">
+		<instruction-set targetVersion="1">
+			<remove-channel id="controlBoilerApproval"/>
+			<remove-channel id="controlProgram"/>
+			<remove-channel id="controlHeatCircProgram0"/>
+			<remove-channel id="controlHeatCircProgram1"/>
+			<remove-channel id="controlHeatCircProgram2"/>
+			<remove-channel id="controlHeatCircProgram3"/>
+			<remove-channel id="controlHeatCircProgram4"/>
+			<remove-channel id="controlHeatCircProgram5"/>
+			<remove-channel id="controlHeatCircProgram6"/>
+			<remove-channel id="controlHeatCircProgram7"/>
+			<remove-channel id="controlHeatCircProgram8"/>
+			<remove-channel id="controlWwHeat0"/>
+			<remove-channel id="controlWwHeat1"/>
+			<remove-channel id="controlWwHeat2"/>
+			<remove-channel id="controlExtraWwHeat0"/>
+			<remove-channel id="controlExtraWwHeat1"/>
+			<remove-channel id="controlExtraWwHeat2"/>
+
+			<add-channel id="boiler-approval" groupIds="control">
+				<type>guntamatic:controlBoilerApproval</type>
```

Comment:
missed this previously, but for channel type's the same `lower-case-hyphen` convention applies and as this is a brekaing change, i would propose to change this too.

Reply:
thanks to co-pilot this change is no longer a major effort ...

Reply:
changed